src/state/achievements/index.ts

```diff
@@ -39,8 +36,6 @@ export const fetchAchievements = (account: string) => async (dispatch: Dispatch)
     dispatch(setAchievements(achievements))
   } catch (error) {
     console.error(error)
-    const title = 'Error fetching achievements'
-    dispatch(push({ id: kebabCase(title), type: toastTypes.DANGER, title }))
```

Comment:
I removed this for 2 reasons:

1. Cannot dispatch toasts anymore.
2. Not sure we should have done this in the first place.